packages/core/src/scope.ts

```diff
@@ -111,6 +118,9 @@ class ScopeClass implements ScopeInterface {
     this._contexts = {};
     this._sdkProcessingMetadata = {};
     this._propagationContext = generatePropagationContext();
+
+    this._flagBuffer = [];
```

Comment:
This needs to be an LRU cache.

Reply:
I think this implementation functions like an LRU cache. At least that's the assumption me and @michellewzhang had?

Reply:
yeah, it's the same way it functions on the frontend for frontend flags. it's implemented as an array with logic that removes duplicates and pushes them to the end of the array.